src/pages/ReimbursementAccount/EnableBankAccount/EnableBankAccount.tsx

```diff
@@ -108,22 +104,20 @@ function EnableBankAccount({reimbursementAccount, user, onBackButtonPress}) {
                             icon={Expensicons.Close}
                             onPress={BankAccounts.requestResetFreePlanBankAccount}
                             wrapperStyle={[styles.cardMenuItem, styles.mv3]}
-                            disabled={Boolean(pendingAction) || !_.isEmpty(errors)}
+                            disabled={Boolean(pendingAction) || isNotEmptyObject(errors)}
```

Comment:
`isNotEmptyObject` is going to be [removed](https://github.com/Expensify/App/issues/33886) soon, please use `!isEmptyObject`